.gitignore

```diff
@@ -14,6 +14,7 @@ node_modules
 /public
 /static
 /vendor
+/php7
```

Comment:
This would also need to be added to `.distignore`

Reply:
Done.